lib/experimental/Information/Tags/StatusTag/index.tsx

```diff
@@ -3,7 +3,7 @@ import { cn } from "@/lib/utils"
 import { forwardRef } from "react"
 import { BaseTag } from "../BaseTag"
 
-type Variant = "neutral" | "info" | "positive" | "warning" | "critical"
+export type Variant = "neutral" | "info" | "positive" | "warning" | "critical"
```

Comment:
We were already exporting this as `StatusVariant` down below, which is a bit more explicit as well naming-wise :)